src/main/steps/common/common.content.ts

```diff
@@ -97,13 +97,13 @@ export const en = {
   cookiesHeading: 'Cookies on',
   cookiesLine1: 'We use some essential cookies to make this service work.',
   cookiesLine2:
-    'We’d also like to use analytics cookies so we can understand how you use the service and make improvements.',
+    'We’d like to set additional cookies so we can remember your settings, understand how people use the service and to improve government services.',
   acceptAnalyticsCookies: 'Accept analytics cookies',
   rejectAnalyticsCookies: 'Reject analytics cookies',
   viewCookies: 'View cookies',
   hideMessage: 'Hide this message',
   cookiesConfirmationMessage:
-    '<p>You can <a class="govuk-link" href="/cookies">change your cookie settings</a> at any time.</p>',
+    '<p class="govuk-body">You can <a class="govuk-link" href="/cookies">change your cookie settings</a> at any time.</p>',
   changeCookiesHeading: 'Change your cookie settings',
   allowAnalyticsCookies: 'Allow cookies that measure website use?',
   useAnalyticsCookies: 'Use cookies that measure my website use',
```

Comment:
Line 126 for Welsh has a TODO. why is the infrastructure missing?

Reply:
Yes Vivek .. Right now added welsh infra for this .